docs-markdown/src/controllers/cleanup-controller.ts

```diff
@@ -344,10 +371,38 @@ function capaitalizationOfMetadata() {
     //     lowerCaseMetadata(file, data, "ms.technology")
     //     lowerCaseMetadata(file, data, "ms.topic")
     // })
+    // var data = readFileSync(file, "utf8")
+    // lowerCaseMetadata(file, data, "ms.author")
+    // lowerCaseMetadata(file, data, "author")
+    // lowerCaseMetadata(file, data, "ms.prod")
+    // lowerCaseMetadata(file, data, "ms.service")
+    // lowerCaseMetadata(file, data, "ms.subservice")
+    // lowerCaseMetadata(file, data, "ms.technology")
+    // lowerCaseMetadata(file, data, "ms.topic")
     walk(workspace.rootPath, function (err: any, res: any) {
         output.appendLine(res);
     })
 }
+function lowerCaseData(file: string, data: any, variable: string) {
+    let regex = new RegExp(`^(${variable}:\\s?)(.*\\s)`, 'm');
+    let metadata = data.match(regex)
+    if (metadata) {
+        let captureParts = regex.exec(data);
+        let value = ''
+        if (captureParts && captureParts.length > 2) {
+            value = captureParts[2].toLowerCase();
+            try {
+                return data.replace(regex, `${variable}: ${value}`)
+            }
+            catch (error) {
+                console.error('Error occurred:', error);
```

Comment:
I'm pretty sure it does, but when the error is logged, does it also log the line number?